src/quo2/components/drawers/action_drawers/view.cljs

```diff
@@ -1,20 +1,21 @@
 (ns quo2.components.drawers.action-drawers.view
-  (:require [quo2.components.icon :as icon]
+  (:require [react-native.core :as rn]
+            [quo2.components.drawers.action-drawers.style :as style]
+            [quo2.components.icon :as icon]
             [quo2.components.markdown.text :as text]
             [quo2.foundations.colors :as colors]
-            [react-native.core :as rn]
-            [quo2.components.drawers.action-drawers.style :as style]))
+            [quo2.theme :as theme]))
```

Comment:
nit: @ilmotta and I noticed that  this can clash with `theme` prop, it's safer if we start doing `[quo2.theme :as quo.theme]`